homepage.php

```diff
@@ -1,73 +1,112 @@
-			<p>Digital music service with millions of high quality copyrighted songs , helping you listen to music, download music, upload and sync my music store on multiple devices.</p>
+			<p >Digital music service with millions of high quality copyrighted songs , helping you listen to music, download music, upload and sync my music store on multiple devices.</p>
 		</div>
+		<!-- Holds the Imges which all be the link to other special playlist-->
 		<div class="content_4">
 			<Space></Space>
 			<div class="images1">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query8.php"><img src = "images/indie.jpg" class="image1"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query8.php">Indie</a></div>
+  				</div>
 			</div>
 			<div class="images2">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query7.php"><img src = "images/R&B.jpg" class="image2"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query7.php">R and B</a></div>
+  				</div>
 			</div>
 			<div class="images3">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query6.php"><img src = "images/pop.jpg" class="image3"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query6.php">Pop</a></div>
+  				</div>
 			</div>
 			<div class="images4">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query10.php"><img src = "images/rock.png" class="image4"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query10.php">Rock</a></div>
+  				</div>
 			</div>
 			<Space></Space>
 			<div class="images5">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query9.php"><img src = "images/easylisten.jpg" class="image5"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query9.php">Easy Listen</a></div>
+  				</div>
 			</div>
 			<div class="images6">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query3.php"><img src = "images/country.png" class="image6"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query3.php">Country</a></div>
+  				</div>
 			</div>
 			<div class="images7">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query4.php"><img src = "images/jazz.png" class="image7"></a>
```

Comment:
26/8/2021: Adding a img that this use to reparent the idea a music type. When the user hover on the img when the overlay techniques will work and it will show the link to the query that store a table of all the music in that specific type of music. Eg. If the img indicate to rock music then when the user hover on that image then it will show up will the text rock and when the user link on the link it will deliver the user to a page of all rock music.